Ontology/Willow/Capability/Parameter/Setpoint/Quantity/TemperatureSetpoint/Air/Zone Air/TargetZoneAirTemperatureSetpoint.json

```diff
@@ -0,0 +1,16 @@
+{
+  "@id": "dtmi:com:willowinc:TargetZoneAirTemperatureSetpoint;1",
+  "@type": "Interface",
+  "displayName": {
+    "en": "Target Zone Air Temperature Setpoint"
+  },
+  "description": {"en": "The desired value associated with target zone air temperature"},
```

Comment:
I'd like to be very specific with how this is defined relative to all of the other zone air temp setpoints. Can we refine this? I think it's definition would include something like "the single user-defined setpoint from which cooling and heating setpoints are derived from by using an offset from the target"

Reply:
Agreed